tests/test_slots.py

```diff
@@ -178,3 +183,10 @@ def test_slots_advance_thread(self):
         with patch.object(SlotsHandler, 'get_local_connection_cursor', Mock(side_effect=Exception)):
             self.s.schedule_advance_slots({'foo': {'bar': 100}})
             self.s._advance.sync_slots()
+
+    @patch.object(Postgresql, 'is_primary', Mock(return_value=False))
+    def test_advance_physical_slots(self):
```

Comment:
`patroni/postgresql/slots.py:389-390` (exception) is not covered
Can we also have the query call's content assertion here? 🥺